pkg/specgen/generate/kube/kube.go

```diff
@@ -431,6 +431,8 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 		s.Annotations[define.InspectAnnotationPublishAll] = publishAll
 	}
 
+	s.Annotations[define.KubeHealthCheckAnnotation] = ""
```

Comment:
setting this to an empty value looks odd, maybe set it to `true` instead

Reply:
Fixed